reducers/selectedDateReducer.js

```diff
@@ -0,0 +1,8 @@
+export const selectedDate = (state = new Date().toDateString(), action) => {
```

Comment:
super smart to set this default value in the store to today's current date. should work very well on page load and help with our initial fetch call